index.html

```diff
@@ -0,0 +1,158 @@
+    rel="stylesheet">
+  <title>Mike's Capstone Project</title>
+</head>
+
+<body>
+  <!-- the navbar (and header) section -->
+  <nav class="mini-nav on-desktop">
+    <ul class="mininav-menu">
+      <li class="mininav-item"><img src="/img/facebook.svg" alt="facebook"></li>
+      <li class="mininav-item"><img src="/img/twitter.svg" alt="twitter"></li>
+      <li class="mininav-menu">English</li>
+      <li class="mininav-menu">My Page</li>
+      <li class="mininav-menu">Logout</li>
+    </ul>
+  </nav>
+  <header id="myheader">
+    <!-- <a class="logo" href="#home">My Logo</a> -->
+    <a class=" logo on-desktop" href="#home"><img src="/img/logo.jpg" class="logo-nav" alt="logo"></a>
+    <nav class="navbar">
+      <ul class="nav-menu">
+        <li class="nav-item">
+          <a href="/about.html" class="nav-link">About</a>
+        </li>
+        <li class="nav-item">
+          <a href="#" class="nav-link">Program</a>
+        </li>
+        <li class="nav-item">
+          <a href="#" class="nav-link">Join</a>
+        </li>
+        <li class="nav-item">
+          <a href="#" class="nav-link">Sponsor</a>
+        </li>
+        <li class="nav-item">
+          <a href="#" class="nav-link">News</a>
+        </li>
+        <li class="nav-item">
+          <a href="#" class="nav-link hg">HG Campaign</a>
+        </li>
+      </ul>
+      <div class="hamburger">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+      </div>
+    </nav>
+  </header>
+  <!-- the Home page -->
+  <section id="home">
+    <div class="main-content">
+      <div class="headline">
+        <h2 class="heading1">"Hello! Sharing World"</h2>
+        <h1 class="main-heading">HopeGrower Gold Global Conference 2023</h1>
+      </div>
+      <p class="main-detail">A joyful celebration believing in the openness and sharing, believing in "Food is the New
+        Gold" creating a positive
+        change with people from over 100 countries is taking place in November, Pakistan</p>
+      <div class="date-venue">
+        <h2 class="dates">2023.11.14(THU) ~ 15(FRI)</h2>
+        <p class="venue">@ National Museum of Pakistan, Art Center Iqbal & more</p>
+      </div>
+    </div>
+  </section>
+  <!-- the Program page -->
+  <section id="program">
+    <div>
+      <h3 class="program-headline">Main Program</h3>
+      <div class="underline"></div>
+    </div>
+    <div class="program-container">
+      <div class="main-program">
+        <div><img src="/img/program_icon_01.png" alt=""></div>
+        <h3 class="program-subheading">
+          Lecture
+        </h3>
+        <p class="program-info">
+          Listen to the speakers from various countries about the message of farming and food.
+        </p>
+      </div>
+      <div class="main-program">
+        <div><img src="/img/program_icon_02.png" alt=""></div>
+        <h3 class="program-subheading">
+          HG Exhibition
+        </h3>
```

Comment:
- [ ] Great job. From the original design, the cards have their separate background color. please implement this.
- [ ] you are missing the underlined text below the cards. kindly add that (see the whole program)
yours
![Screenshot (1076)](https://github.com/Mike111222/CapstoneProject/assets/65298048/e738f9ca-fd15-4640-b222-ce9026af29a4)
original
![Screenshot (1011)](https://github.com/Mike111222/CapstoneProject/assets/65298048/fe838291-c23a-4095-a746-8f2cf2dc2abe)